### DISCUSSION TITLE
Vercel Build Logs

### DISCUSSION THREAD

**Discussion by perperj-super** [vercel/community#2786] [Help]
### Summary

Hello Vercel team!

I deployed my Next.js application and I got the deployment failure.

Maybe a couple of months ago, I could see the build logs so I could easily fix the errors and redeploy it.

But at the moment, I can't see any build logs.

Could you let me know how to see the build logs?

Thanks in advance

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/101011021/9fec2476-b3ea-4c53-8079-939ab57a6f94)

I clicked the `Expand All` button, but I can't see any logs there

![image](https://github.com/vercel/community/assets/101011021/dc4b21a6-8e77-411c-a32b-54d8b19bd417)

**Comment by Lewitje** [comment]
I'm having the same issue, I can't see any logs on any projects

**Reply by perperj-super** [reply]
@Lewitje, I needed to wait for a long time and finally could see the logs

**Comment by zacBkh** [comment]
Same issue here.. #2895
However, it works perfectly on mobile!

**Comment by Lewitje** [comment]
This is still an issue
